app/main.py

```diff
@@ -1 +1,21 @@
-# write your code here
+from app.cafe import Cafe
+from app.errors import (NotWearingMaskError,
+                        NotVaccinatedError,
+                        OutdatedVaccineError)
+
+
+def go_to_cafe(friends, cafe) -> str:
+    total = 0
+
+    for i in range(len(friends)):
```

Comment:
Will be much better to iterate over elements directly, just `for friend in friends:` it saves one variable, you get rid of length calculation and `friends` can be a generator (for example it can be a generator that returns objects one by one from the database, file and etc).